src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
> Then fix the whole file, if you think it's that important.

I'll prefer to wait for druntime devs to chain in. It's also possible they request to change my style to `static if` as they like it too just like you and I'm the minority.